tests/integration/transformers/test_categorical.py

```diff
@@ -47,7 +47,7 @@ def test_clip_interval(self):
     def test__reverse_transform(self):
         """Test the ``reverse_transform``."""
         # Setup
-        data = pd.DataFrame({'column_name': [1, 2, 3, 2, 2, 1, 3, 3, 2]})
+        data = pd.DataFrame({'column_name': [-1, 0, 1, 2, 3, 2, 2, 1, 3, 3, 2, -11, 0]})
```

Comment:
I think since this test runs end to end it won't actually test `reverse_transform` on negative values since it runs against the transformed values on line 57. After being transformed all the negative values are removed. I think we can just add this as a unit test or update the existing one